support/mock/mockrt_state_test.go

```diff
@@ -52,7 +52,8 @@ func (s State) MarshalCBOR(w io.Writer) error {
 func (s State) UnmarshalCBOR(r io.Reader) error {
 	var cint cbg.CborInt
 	err := cint.UnmarshalCBOR(r)
-	s.Value = int64(cint)
+	// should the state have been passed-by-reference or something? why is this here?
```

Comment:
Might be a bug.  File an issue, remove this comment and uncomment out the potential bug.  Then we can look into testing this thing.

This seems to only be used in TestIllegalStateModifications which isn't obviously exercising correct cbor marshal unmarshal.